src/build/arg/mod.rs

```diff
@@ -95,8 +100,17 @@ impl<'help> Arg<'help> {
     }
 
     /// Get the help specified for this argument, if any
+    #[deprecated(
+        since = "3.0.0",
+        note = "Please use `get_about` method instead"
+    )]
     pub fn get_help(&self) -> Option<&str> {
-        self.help
+        self.about
+    }
```

Comment:
This method did not existed in 2.x. It was introduced literally a few commits ago, so there's no point in deprecating - users never had a chance to see it. Just rename it to `get_about`.